docs/pages/components/ActionMenu.mdx

```diff
@@ -416,6 +416,11 @@ function MixedSelectionExample() {
       name: 'open',
       type: 'boolean',
       description: 'Controls whether the action menu is open. Only necessary for controlling action menu state.'
+    },
+    {
+      name: 'portal',
+      type: ['React.Ref', 'HTMLElement'],
+      description: 'The parent element to place the ActionMenu in.'
```

Comment:
Feel free to ignore - 

Is it worth linking to React documentation on [Portals](https://react.dev/reference/react-dom/createPortal) at all here?

Reply:
We don't have an easy way to link in to documentation with the way this is built right now.